scss/components/_nav.scss

```diff
@@ -0,0 +1,18 @@
+.nav-link {
+  color: $body-color;
+
+  @include media-breakpoint-up(lg) {
+    padding: $nav-link-padding-y-lg $nav-link-padding-x-lg;
+  }
+
+  &.active,
+  &:hover {
+    color: $primary;
+    text-decoration: underline;
+  }
+
+  &.link-search:hover {
```

Comment:
we don't need this specifity, .link-search is enough to get the style. Please put it outside